lib/player_module.rb

```diff
@@ -0,0 +1,14 @@
+require "human"
+require "computer"
+
+module Player
+  def self.generate(type, board)
+    if type == 'hh'
+      players = [Human.new("X"), Human.new("O")]
+    elsif type == "hc"
+      players = [Human.new("X"), Computer.new("O", board)]
+    end
+
```

Comment:
i'd lose this line

Reply:
I've used a ternary operator instead to shorten it.